### DISCUSSION TITLE
Search component in showcase - primeNG

### DISCUSSION THREAD

**Discussion by aparzi** [primefaces/community#1273] [PrimeNG] [answered]
When I search for something in the search component in the topbar, using an older version of primeNG such as v15-lts, I am redirected to the attusle version.  you might think about searching in the version selected by the user.  If so, I could work on it myself, if possible.  Thank you

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for reporting us, we'll move lts versions to the new domain to fix this issue in the near future.

**Reply by aparzi** [reply]
Hi @cetincakiroglu,
thanks. If you need help I'm happy to do it, thanks.
